src/engine/perf.h

```diff
@@ -19,13 +19,13 @@
 
 /*!
  * Copyright (c) 2017 by Contributors
- * \file test_perf.h
- * \brief operator unit test utility functions
+ * \file perf.h
+ * \brief operator performance classes and functions
```

Comment:
Actually, I didnt end up using perf.h, so moving this back to unit test directory...